frontend-pages/profile-page.php

```diff
@@ -36,11 +36,14 @@ function ssv_profile_page_setup($content)
 
     if (isset($_GET['view']) && $_GET['view'] == 'pdf') {
         require_once(ABSPATH . 'wp-content/plugins/ssv-frontend-members/include/fpdf/SSV_FPDF.php');
+        if (isset($_GET['user_id'])) {
+            $frontendMember = FrontendMember::get_by_id($_GET['user_id']);
+        } else {
+            $frontendMember = FrontendMember::get_current_user();
+        }
         $pdf = new SSV_FPDF();
-        $pdf->AddPage();
-        $pdf->SetFont('Arial','B',16);
-        $pdf->Cell(40,10,'Hello World!');
-        $pdf->Output();
+        $pdf->build($frontendMember);
+        $pdf->Output('F', ABSPATH . 'wp-content/plugins/ssv-frontend-members/exported-profiles/' . str_replace(' ', '_', $frontendMember->display_name) . '.pdf');
```

Comment:
That was the idea but because of the WordPress security you can't get any user data from the plugin directly, and you can't stream the PDF from the WordPress environment due to WordPress already echoing code (theme, base layout etc.).